cha/sota/sota_app.tex

```diff
@@ -9,53 +9,52 @@ \section{Android Apps}\label{sec:sota_apps}
 \end{itemize}
 
 These apps are found by searching sources like Google\footnote{https://www.google.dk with search terms ``android sync music playback''}, YouTube\footnote{https://www.youtube.com/}, App Store\footnote{https://itunes.apple.com/dk}, and Google Play Store\footnote{https://play.google.com/store} and from recommendations on forums.
-A criteria for an app was that the it must not be depreciated or abandoned by its developers.
-The way to estimate this was done by looking at the last update date on Google Play Store and App Store.
-For instance if an app had not been updated for a long time, e.g. since 2013 and it do not support new devices, it is probably abandoned.
+An additional criteria for an app to be considered here is that the app must not be abandoned by the developers,
+which we defined as being without an update since, at least, 2013, and being incompatible with new devices.
 
-To clarify, Google Play Store and App Store are the official place to install or buy apps for Android and iOS devices respectively.
+To clarify, Google Play Store and the App Store are the official places to install or buy apps for Android and iOS devices respectively.
 
-The three apps all use a master/slave connection.
-All three apps use different terms for their setup, for clarification we generalise these terms.
-We call the device which selects the music and streams it, the master.
-The devices which connects to the master (SoundSeeder calls it ``Speaker''), are referred to as slaves.
+The three apps all use a master/slave connection and use different terms for their setup.
+For clarification we generalise these terms. 
+We call the device which selects the music and streams it, the master, and the devices which connects to the master (SoundSeeder calls it ``Speaker''), are referred to as slaves.
 
 \subsection{SoundSeeder}\label{subsec:soundseeder}
 SoundSeeder\footnote{http://soundseeder.com} is an app made by JekApps.
-The used version of the app is 1.6.5.
+The current version as of writing is 1.6.5.

```

Comment:
Har du tænkt dig at tjekke om dette er korrekt inden i aflevere? for ellers virker det andet bedre